app.js

```diff
@@ -1,28 +1,24 @@
+//packages
 var Discord = require("discord.js")
-var bot = new Discord.Client();
+var client = new Discord.Client();
+var ffmpeg = require("ffmpeg-binaries")
+//configuration stuff
+const {apikey} = require("./config");
+const {prefix} = require("./config");
+client.login(apikey);
 
-const {apikey} = require("./api-key");
 
+//commands
+const {join} = require("./commands/basic/join");
+const {music} = require("./commands/music/music");
 
-var prefix = '>';
+//prefix to start commands with
 
-bot.on("ready", () => {
+
+client.on("ready", () => {
     console.log("Bot Launched... ")
```

Comment:
logging lib might be helpful -- to differentiate debug and Info

Reply:
Ill think about using one for my next project